setup.py

```diff
@@ -30,6 +30,8 @@ def read(filename):
     packages=['crate.qa'],
     install_requires=[
         'cr8==0.12-master',
+        'Cython',
+        'asyncpg'
```

Comment:
on my machine this runs into `SQLParseException: line 2:1: mismatched input 'CLOSE' expecting <EOF>
` due to the unreleased patch that we contributed.

Seems like travis is also not picking up the test?

Reply:
this is odd